### PR TITLE
Revert GraphQL via ZMQ option

### PR DESCRIPTION
**Pull request by felixvd**
Reverts #124 because we do this in a separate repository now.

cc @superfashi

### CODE REVIEW

**Comment by superfashi** [comment]
Sorry for pushing to the wrong branch...

**Comment by felixvd** [comment]
Ping @ziyan

**Comment by cielavenir** [comment]
btw this can be merged at any timing; controllerclientcpppy can be still built (it just dynamically loses ExecuteGraphQueryZmq binding) thanks to SFINAE.

File: src/mujinzmq.cpp [hunk 1]
```diff
@@ -495,7 +494,6 @@ void ZmqServer::_InitializeSocket(boost::shared_ptr<zmq::context_t> context)
 void ZmqServer::_DestroySocket()
 {
     if (!!_socket) {
-        _socket->setsockopt(ZMQ_LINGER, 0);
```
Comment (cielavenir) [inline src/mujinzmq.cpp:498]: @felixvd oh, should this be still useful for binpickingtask?

Comment (felixvd) [reply src/mujinzmq.cpp]: Whoops. Removed this.

**Pull request merged**